IBX-5222: Work with drafts (#298)
ezsystems/user-documentation: docs/content_management/workflow_management/work_with_versions.md

Reviewer: ```suggestion
description: Perform various tasks on content item versions, as part of editorial workflow or when comparing edits from different users.
month_change: true
```

In [[= product_name =]], content items can have one published [version](../content_versions.md), 
and several draft and archived versions.
You can perform various tasks on content item versions, either to advance them through the workflow or compare edits from different users.

## Edit drafts

There are several ways in which you can start editing an existing draft:

- by picking a draft for editing from the content item's details screen, **Versions** tab, the **Open drafts** list.
You may need to [disable the Focus mode](../../getting_started/discover_ui.md#disable-focus-mode) first. 

![Open drafts list](img/open_drafts.png "Open drafts list")

- by trying to edit a content item that already has open drafts.

> You then see a [**Draft conflict**](content_versions.md#draft-conflicts) screen, where you can pick a draft for editing, or create a new one.

Author: ```suggestion
You then see a [**Draft conflict**](content_versions.md#draft-conflicts) screen, where you can pick a draft for editing, or create a new one.
The new draft is based on the published version of the content item and does not contain changes from other drafts.
```

In [[= product_name =]], content items can have one published [version](../content_versions.md), 
and several draft and archived versions.
You can perform various tasks on content item versions, either to advance them through the workflow or compare edits from different users.

## Edit drafts

There are several ways in which you can start editing an existing draft:

- by picking a draft for editing from the content item's details screen, **Versions** tab, the **Open drafts** list.
You may need to [disable the Focus mode](../../getting_started/discover_ui.md#disable-focus-mode) first. 

![Open drafts list](img/open_drafts.png "Open drafts list")

> - by trying to edit a content item that already has open drafts.

Author: ```suggestion
- by editing a content item that already has open drafts.
```
